traffic_portal/app/src/common/modules/form/iso/form.iso.tpl.html

```diff
@@ -28,163 +28,162 @@
+                <label for="hostName" class="control-label col-md-2 col-sm-2 col-xs-12">Hostname *</label>
                 <div class="col-md-10 col-sm-10 col-xs-12">
-                    <input name="hostName" type="text" class="form-control" ng-model="iso.hostName" required autofocus>
+                    <input id="hostName" name="hostName" type="text" class="form-control" ng-model="iso.hostName" required>
                     <small class="input-error" ng-show="hasPropertyError(isoForm.hostName, 'required')">Required</small>
                     <span ng-show="hasError(isoForm.hostName)" class="form-control-feedback"><i class="fa fa-times"></i></span>
                 </div>
             </div>
             <div class="form-group" ng-class="{'has-error': hasError(isoForm.domainName), 'has-feedback': hasError(isoForm.domainName)}">
-                <label class="control-label col-md-2 col-sm-2 col-xs-12">Domain *</label>
+                <label for="domainName" class="control-label col-md-2 col-sm-2 col-xs-12">Domain *</label>
                 <div class="col-md-10 col-sm-10 col-xs-12">
-                    <input name="domainName" type="text" class="form-control" ng-model="iso.domainName" required autofocus>
+                    <input id="domainName" name="domainName" type="text" class="form-control" ng-model="iso.domainName" required>
                     <small class="input-error" ng-show="hasPropertyError(isoForm.domainName, 'required')">Required</small>
                     <span ng-show="hasError(isoForm.domainName)" class="form-control-feedback"><i class="fa fa-times"></i></span>
                 </div>
             </div>
-            <div class="form-group" ng-class="{'has-error': hasError(isoForm.dhcp), 'has-feedback': hasError(isoForm.dhcp)}">
-                <label class="control-label col-md-2 col-sm-2 col-xs-12">DHCP *</label>
-                <div class="col-md-10 col-sm-10 col-xs-12">
-                    <select name="dhcp" class="form-control" ng-model="iso.dhcp" ng-options="x.value as x.label for x in falseTrue" required>
-                        <option value="">Select...</option>
-                    </select>
-                    <small class="input-error" ng-show="hasPropertyError(isoForm.dhcp, 'required')">Required</small>
+            <div class="form-group">
+                <label for="dhcp" class="control-label col-md-2 col-sm-2 col-xs-12" style="padding-top: 0">DHCP</label>
+                <div class="col-md-10 col-sm-10 col-xs-12" style="width: auto">
+                    <input type="checkbox" id="dhcp" name="dhcp" ng-model="iso.dhcp"/>
                 </div>
             </div>
-            <div class="form-group" ng-show="!isDHCP(iso)" ng-class="{'has-error': hasError(isoForm.ipAddress), 'has-feedback': hasError(isoForm.ipAddress)}">
+            <div class="form-group" ng-if="!iso.dhcp" ng-class="{'has-error': hasError(isoForm.ipAddress), 'has-feedback': hasError(isoForm.ipAddress)}">
```

Comment:
this is getting screwed up by:

```
iso.dhcp = iso.dhcp ? "yes" : "no";
```

and yeah, really dump api design.

Reply:
Should be fixed.